package.json

```diff
@@ -47,7 +47,7 @@
     "postcss-loader": "4.0.4",
     "sass-loader": "8.0.2",
     "svg-inline-loader": "0.8.2",
-    "svg-url-loader": "3.0.3",
+    "svg-url-loader": "6.0.0",
```

Comment:
versions 4 and 5 were just because the underlying dependency on `file-loader` underwent a major bump - but not API changes.  Version 6 sets a default `stripDeclarations: true` option which should be fine for us.

https://github.com/bhovhannes/svg-url-loader/releases/tag/v6.0.0